### PR TITLE
Image upload to S3 using axios, implemented with React front-end.

### PR DESCRIPTION
**Pull request by zhangbrady4**
https://github.com/CloudClub-uoft/GuessTheLocation/assets/127697715/85307944-16e8-4a8e-bb39-901ca5a0bce9

### CODE REVIEW

File: server/config/CloudClub.png
Comment (qingyuan-wu) [inline server/config/CloudClub.png:1]: Can we remove this image? Seems unused

File: server/config/s3setup.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+/*
+s3Client.upload(params, function (err, data) {
+    if (err) {
+        console.log(err);
+    }
+    else
+        console.log("\nUploaded Object:\n", data);
+})
+*/
```
Comment (qingyuan-wu) [inline server/config/s3setup.js:28]: Is this used? If not can we remove?

File: server/server.js [hunk 2]
```diff
@@ -1,7 +1,15 @@
 const express = require('express');
 require("dotenv").config();
 const app = express();
-//const cors = require('cors')
+const fileUpload = require("express-fileupload")
+app.use(fileUpload())
+
+// const s3client = require("./config/s3")
+const s3Client = require("./config/s3setup")
+app.set("view engine", "ejs")
```
Comment (qingyuan-wu) [inline server/server.js:9]: want to remove this as we're not using ejs

File: server/server.js [hunk 3]
```diff
@@ -1,7 +1,15 @@
 const express = require('express');
 require("dotenv").config();
 const app = express();
-//const cors = require('cors')
+const fileUpload = require("express-fileupload")
+app.use(fileUpload())
+
+// const s3client = require("./config/s3")
+const s3Client = require("./config/s3setup")
```
Comment (qingyuan-wu) [inline server/server.js:8]: can we remove unused code?

File: server/server.js [hunk 4]
```diff
@@ -16,10 +24,10 @@ const authRouter = require('./routes/auth');
 const guessRouter = require('./routes/guesses');
 app.use("/users", usersRouter);
 app.use("/posts", postsRouter);
-app.use("/auth", authRouter);
+app.use("/uath", authRouter);
```
Comment (qingyuan-wu) [inline server/server.js:27]: hmm typo here?

File: server/views/uploaddone.ejs [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
```
Comment (qingyuan-wu) [inline server/views/uploaddone.ejs:1]: Let's remove the entire `views` folder since we're not using ejs?

File: client/src/pages/WritePost.jsx [hunk 6]
```diff
@@ -1,29 +1,89 @@
...
           <span>
             <b> Visibility: </b> Public
           </span>
+
+          {/** 
           <input style={{display:"none"}} type="file" id="file"/>
           <label htmlFor="file">Upload Image</label>
+          */}
+
+          {/* <form action="/postupload" method="POST" encType="multipart/form-data"> */}
```
Comment (qingyuan-wu) [inline client/src/pages/WritePost.jsx:68]: Let's remove unused code (same as above/below)

**Review by qingyuan-wu — CHANGES_REQUESTED** [review]

**Comment by zhangbrady4** [comment]
Removed code for EJS and other unused code as requested in latest commit

**Pull request merged**